GraphQL-TSC.md

```diff
@@ -240,14 +240,14 @@ GraphQL projects should use one of the following notices on their websites, as a
 > ```
 > ---
 >
-> Copyright Joint Development Foundation Projects, LLC, GraphQL Series.  
+> Copyright Joint Development Foundation Projects, LLC, GraphQL Series.
```

Comment:
Should this and the below newline be kept?
```suggestion
> Copyright Joint Development Foundation Projects, LLC, GraphQL Series.  
```

Reply:
Let me fix this - my IDE removes trailing space. I'll do a formatting pass and rebase